packages/insomnia-inso/src/commands/generate-config.test.ts

```diff
@@ -23,6 +23,18 @@ const mockConversionResult: KongForKubernetesResult = {
   warnings: [],
 };
 
+const mockDeclarativeConversionResult: DeclarativeConfigResult = {
+  documents: [
+    // @ts-expect-error -- TSCONVERSION the tests seem to suggest that this is valid, yet it is not allowed by the types.
+    {
+      '_format_version': '1.1',
+    },
```

Comment:
```suggestion
    {
      '_format_version': '1.1',
      services: [],
      upstreams: [],
    },
```

This needs to be coupled with the assertion change suggestion further down